README.md

```diff
@@ -7,25 +7,19 @@
 [![Twitter](https://libre-tube.github.io/assets/tw-widget.svg)](https://twitter.com/libretube)
 [![Reddit](https://libre-tube.github.io/assets/rd-widget.svg)](https://www.reddit.com/r/Libretube/)
  
-[<img src="https://libre-tube.github.io/assets/fdrload.png" alt="Get it on F-Droid" height="80">](https://f-droid.org/en/packages/com.github.libretube/)
-[<img src="https://libre-tube.github.io/assets/izzyload.png" alt="Get it on IzzyOnDroid" height="80">](https://apt.izzysoft.de/fdroid/index/apk/com.github.libretube)
-[<img src="https://libre-tube.github.io/assets/ghload.png" alt="Get it on GitHub" height="80">](https://github.com/libre-tube/LibreTube/releases/latest)
-[<img src="https://libre-tube.github.io/assets/tgload.png" alt="Get it on GitHub" height="80">](https://t.me/LibreTube)
+[<img src="https://libre-tube.github.io/assets/fdrload.png" alt="Get it on F-Droid" width="45%">](https://f-droid.org/en/packages/com.github.libretube/) [<img src="https://libre-tube.github.io/assets/izzyload.png" alt="Get it on IzzyOnDroid" width="45%">](https://apt.izzysoft.de/fdroid/index/apk/com.github.libretube) [<img src="https://libre-tube.github.io/assets/ghload.png" alt="Get it on GitHub" width="45%">](https://github.com/libre-tube/LibreTube/releases/latest) [<img src="https://libre-tube.github.io/assets/tgload.png" alt="Get it on GitHub" width="45%">](https://t.me/LibreTube)
```

Comment:
could u make them a bit smaller (like 30%)? they look way too big on desktop

Reply:
Cuz that way, both layouts' issue gets solved

@XelXen have a look sir

Reply:
U dont trust me?